### PR TITLE
[DNM] Test

### PR DESCRIPTION
**Pull request by creydr**
To debug current CI issues

### CODE REVIEW

**Comment by creydr** [comment]
@pierDipi 
we are getting 
```
FATAL	environment/magic.go:241	failed to install CA certificates and issuer: the server could not find the requested resource	{"test": "TestContainerSourceWithTLS"}
```
It is the same issue we had in upstreams, with the TLS tests in eventing-istio. Even we skip the TLS tests, when istio is enabled, they fail, because in the environment setup, we tell them to use eventshub with TLS: https://github.com/knative/eventing/blob/6911db0b3592c6f043455b9be9d3052218cbe05b/test/rekt/container_source_test.go#L99

I am wondering, if we can/should install cert-manager for the e2e tests too (we have it in third_party/eventing/third_party/cert-manager) :thinking:

**Comment by pierDipi** [comment]
> @pierDipi we are getting
> 
> ```
> FATAL	environment/magic.go:241	failed to install CA certificates and issuer: the server could not find the requested resource	{"test": "TestContainerSourceWithTLS"}
> ```
> 
> It is the same issue we had in upstreams, with the TLS tests in eventing-istio. Even we skip the TLS tests, when istio is enabled, they fail, because in the environment setup, we tell them to use eventshub with TLS: https://github.com/knative/eventing/blob/6911db0b3592c6f043455b9be9d3052218cbe05b/test/rekt/container_source_test.go#L99
> 
> I am wondering, if we can/should install cert-manager for the e2e tests too (we have it in third_party/eventing/third_party/cert-manager) thinking

It seems reasonable to install cert-manger

File: openshift/e2e-tests.sh [hunk 1]
```diff
@@ -7,4 +7,14 @@ export GOPATH=/tmp/go
...
 
+pushd "${repo_root_dir}/third_party/eventing"
+echo "Apply eventing submodule patches"
+git apply -v ../../openshift/submodule-patches/eventing/*
+
+echo "Install cert-manager"
+kubectl apply -f third_party/cert-manager/01-cert-manager.crds.yaml || exit 1
+kubectl apply -f third_party/cert-manager/02-cert-manager.yaml || exit 1
+
+popd
```
Comment (pierDipi) [inline openshift/e2e-tests.sh:18]: Can we use SO makefile commands?

Comment (creydr) [reply openshift/e2e-tests.sh:18]: Ah. Now I understand, why we didn't have to install certmanager manually for the m/s eventing e2e tests...
(somehow didn't see it initially in the SO Makefile :facepalm: )

**Pull request closed without merging**